test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/ListPackageCommandRunnerTests.cs

```diff
@@ -17,44 +11,44 @@ public class ListPackageCommandRunnerTests
         public class TopLevelPackagesFilterForOutdated
         {
             [Fact]
-            public async Task FiltersAutoReferencedPackages()
+            public void FiltersAutoReferencedPackages()
             {
                 // Arrange
-                var filter = ListPackageCommandRunner.TopLevelPackagesFilterForOutdated;
-                var installedPackageReference = CreateInstalledPackageReference(autoReference: true);
+                var filter = ListPackageHelper.TopLevelPackagesFilterForOutdated;
+                var installedPackageReference = ListPackageTestHelper.CreateInstalledPackageReference(autoReference: true);
 
                 // Act
-                var result = await filter.Invoke(installedPackageReference);
+                var result = filter.Invoke(installedPackageReference);
```

Comment:
I know you were just fixing tests to compile/pass, but [coding guidelines n. 10](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md) is to use var only when it's obvious what the type is. None of these are obvious to me what the types are. This is repeated throughout the file.

Reply:
This is a good opportunity to conform the code to guidelines, so I'll make those changes.